pkg/kepctl/query.go

```diff
@@ -54,7 +54,7 @@ type QueryOpts struct {
 }
 
 // Validate checks the args and cleans them up if needed
```

Comment:
Q (since you're in this file already :smile: ): is this comment^^ correct?

I don't really see the func cleaning them up?

Reply:
Yeah, we can see function is not using the args now.  Removing this comment. Thanks for the review.